Library/Formula/gl2ps.rb

```diff
@@ -14,8 +14,10 @@ class Gl2ps < Formula
 
   depends_on "cmake" => :build
   depends_on "libpng"
+  depends_on :x11 => :optional
 
   def install
+    inreplace "CMakeLists.txt", "find_package(GLUT)", "" if build.without? "x11"
```

Comment:
@xu-cheng ah, yeah you're right. When X11 is not available, it ends up like this:
```
iMac-TMP:~ joe$ brew linkage gl2ps
System libraries:
  /System/Library/Frameworks/Cocoa.framework/Versions/A/Cocoa
  /System/Library/Frameworks/GLUT.framework/Versions/A/GLUT
  /System/Library/Frameworks/OpenGL.framework/Versions/A/OpenGL
  /usr/lib/libSystem.B.dylib
  /usr/lib/libz.1.dylib
Homebrew libraries:
  /usr/local/opt/libpng/lib/libpng16.16.dylib (libpng)
```
So we should probably do that regardless of whether X11 is available.